spec/features/daily_digest_spec.rb

```diff
@@ -1,12 +1,12 @@
 RSpec.describe "Daily digests", type: :request do
-  let(:list_one_topic_id) { "0eb5d0f0-d384-4f27-9da8-3f9e9b22a820" }
+  let(:list_one_taxon_id) { "0eb5d0f0-d384-4f27-9da8-3f9e9b22a820" }
   let(:list_two_taxon_id) { "6416e4e0-c0c1-457a-8337-4bf8ed9d5f80" }
 
   let(:subscriber_list_one) do
     create_subscriber_list(
       title: "Subscriber list one",
       links: {
-        topics: { any: [list_one_topic_id] },
+        taxons: { any: [list_one_taxon_id] },
```

Comment:
SubscriberLists use the key taxon_tree

eg
```
  id: 1447,
  title: "Education, training and skills",
  created_at: Thu, 30 Mar 2017 17:06:26.031943000 BST +01:00,
  updated_at: Wed, 20 Jan 2021 09:07:44.083612000 GMT +00:00,
  document_type: "",
  tags: {},
  links: {"taxon_tree"=>{"any"=>["c58fdadd-7743-46d6-9629-90bb3ccc4ef0"]}},
  email_document_supertype: "",
  government_document_supertype: "",
  signon_user_uid: nil,
  slug: "education-training-and-skills",
  url: "/education",
  tags_digest: nil,
  links_digest: "1c115d268a3d4a4dfb7875470573d76793ddc645cadbb3cd3ed34188da2d4237",
  content_id: nil,
  description: nil,
  last_audited_at: Wed, 24 Apr 2024 13:31:57.000000000 BST +01:00>
```
See on line 18 below

Reply:
Fixed